main.go

```diff
@@ -166,26 +168,23 @@ func main() {
 		})
 	}
 
+	tlsConfig := opts.GetTLSConfigPath()
+	promlogConfig := &promlog.Config{}
```

Comment:
We should stick to a unified logging mechanism. We already have a `promLogger` type that extends a Logging interface. We can make `promLogger` extend the new interface exposed in `web.ListenAndServe`